src/test/java/uk/gov/hmcts/reform/jobscheduler/FixtureData.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.hmcts.reform.jobscheduler;
+
+import feign.FeignException;
+import org.springframework.http.HttpStatus;
+import uk.gov.hmcts.reform.authorisation.exceptions.AbstractAuthorisationException;
+
+import java.util.Collections;
+
+public final class FixtureData {
```

Comment:
is this different from existing `SampleData`?

Reply:
kind of. at first i have put in `SampleData` but then figured - this is not actually a sample, it's more of a setup, a fixture of sorts. therefore a new class to not confuse with logical elements within one and another